### ISSUE TITLE
couple of things after CE

### ISSUE THREAD

**Issue by JaninaPi** [raspberrypilearning/c3-wolfram-facial-recognition#4]
- Since this project asks learners to search the internet for images, it would be good to add something about e-safety in this context. I think there might be existing projects you can copy-paste from and there is also this ingredient: https://github.com/raspberrypilearning/images-permissions-to-use

- I've copy-pasted a lot of step 3 into step 4, bc step 4 didn't actually have instructions for how to use the imported image in the classifier. Please double-check that the instructions work as they are.

**Comment by PhilipHarney** [comment]
I've added a little text to step 2 to address the potential online safety considerations. The learner doesn't actually see the images retrieved if they code it correctly, but tinkering with the code or making some very specific mistakes could reveal them. Good catch!

As for the tasks you added to step 4, I removed the second one but left the first in place. I don' think there's a way to create a single button that does everything with an uploaded file. I also don't want to double-up too much with the next step.
